protocol/test/foundry/utils/TestHelper.sol

```diff
@@ -0,0 +1,388 @@
+/**
+ * SPDX-License-Identifier: MIT
+ **/
+pragma solidity =0.7.6;
+pragma abicoder v2;
+
+
+import "forge-std/console.sol";
+import "forge-std/Test.sol";
+import "./Strings.sol";
+
+import {Utils} from "test/foundry/utils/Utils.sol";
+
+// Diamond setup
+import {Diamond} from "~/beanstalk/Diamond.sol";
+import {IDiamondCut} from "~/interfaces/IDiamondCut.sol";
+import {MockInitDiamond} from "~/mocks/MockInitDiamond.sol";
+
+/// Modules
+// Diamond
+import {DiamondCutFacet} from "~/beanstalk/diamond/DiamondCutFacet.sol";
+import {DiamondLoupeFacet} from "~/beanstalk/diamond/DiamondLoupeFacet.sol";
+import {PauseFacet} from "~/beanstalk/diamond/PauseFacet.sol";
+import {OwnershipFacet} from "~/beanstalk/diamond/OwnershipFacet.sol";
+
+// Silo
+import {MockSiloFacet} from "~/mocks/mockFacets/MockSiloFacet.sol";
+import {BDVFacet} from "~/beanstalk/silo/BDVFacet.sol";
+import {ConvertFacet} from "~/beanstalk/silo/ConvertFacet.sol";
+import {WhitelistFacet} from "~/beanstalk/silo/WhitelistFacet.sol";
+
+// Field
+import {MockFieldFacet} from "~/mocks/mockFacets/MockFieldFacet.sol";
+import {MockFundraiserFacet} from "~/mocks/mockFacets/MockFundraiserFacet.sol";
+
+// Farm
+import {FarmFacet} from "~/beanstalk/farm/FarmFacet.sol";
+import {CurveFacet} from "~/beanstalk/farm/CurveFacet.sol";
+import {TokenFacet} from "~/beanstalk/farm/TokenFacet.sol";
+
+/// Ecosystem
+import {BeanstalkPrice} from "~/ecosystem/price/BeanstalkPrice.sol";
+
+/// Mocks
+import {MockConvertFacet} from "~/mocks/mockFacets/MockConvertFacet.sol";
+import {MockMarketplaceFacet} from "~/mocks/mockFacets/MockMarketplaceFacet.sol";
+import {MockSeasonFacet} from "~/mocks/mockFacets/MockSeasonFacet.sol";
+import {MockFertilizerFacet} from "~/mocks/mockFacets/MockFertilizerFacet.sol";
+import {MockToken} from "~/mocks/MockToken.sol";
+import {MockUnripeFacet} from "~/mocks/mockFacets/MockUnripeFacet.sol";
+import {Mock3Curve} from "~/mocks/curve/Mock3Curve.sol";
+import {MockUniswapV3Pool} from "~/mocks/uniswap/MockUniswapV3Pool.sol";
+import {MockUniswapV3Factory} from "~/mocks/uniswap/MockUniswapV3Factory.sol";
+import {MockCurveFactory} from "~/mocks/curve/MockCurveFactory.sol";
+import {MockCurveZap} from "~/mocks/curve/MockCurveZap.sol";
+import {MockMeta3Curve} from "~/mocks/curve/MockMeta3Curve.sol";
+import {MockWETH} from "~/mocks/MockWETH.sol";
+
+import "~/beanstalk/AppStorage.sol";
+import "~/libraries/Decimal.sol";
+import "~/libraries/LibSafeMath32.sol";
+import "~/libraries/Token/LibTransfer.sol";
+
+import "~/C.sol";
+
+// FIXME: 
+// currently with all tests, we deploy every facet, even if its unused
+// 
+abstract contract TestHelper is Test {
```

Comment:
Dangling FIXME comment.

This seems to be a duplicate of InitDiamondDeployer, which one is correct?